certif/app/services/locale.js

```diff
@@ -1,19 +1,25 @@
 import Service, { inject as service } from '@ember/service';
 import config from 'pix-certif/config/environment';
 
-const { COOKIE_LOCALE_LIFESPAN_IN_SECONDS } = config.APP;
-
 export const FRENCH_INTERNATIONAL_LOCALE = 'fr';
 export const ENGLISH_INTERNATIONAL_LOCALE = 'en';
 export const FRENCH_FRANCE_LOCALE = 'fr-FR';
 export const DEFAULT_LOCALE = FRENCH_INTERNATIONAL_LOCALE;
 
+const { COOKIE_LOCALE_LIFESPAN_IN_SECONDS } = config.APP;
+const supportedLanguages = [FRENCH_INTERNATIONAL_LOCALE, ENGLISH_INTERNATIONAL_LOCALE];
```

Comment:
Il me semble mieux que `supportedLanguages`, étant une constante, soit indiquée comme telle :

```suggestion
export const DEFAULT_LOCALE = FRENCH_INTERNATIONAL_LOCALE;
const SUPPORTED_LANGUAGES = [FRENCH_INTERNATIONAL_LOCALE, ENGLISH_INTERNATIONAL_LOCALE];

const { COOKIE_LOCALE_LIFESPAN_IN_SECONDS } = config.APP;
```

C'est fait de cette manière dans #6126, même si cela n'a pas encore été validé en standard d'équipe.

Reply:
c'est fait 😄